irohad/ordering/impl/ordering_service_transport_grpc.hpp

```diff
@@ -53,6 +54,8 @@ namespace iroha {
       std::weak_ptr<iroha::network::OrderingServiceNotification> subscriber_;
       std::shared_ptr<network::AsyncGrpcClient<google::protobuf::Empty>>
           async_call_;
+      std::unique_ptr<shared_model::interface::TransactionBatchFactory>
```

Comment:
`shared_ptr` might work better since this factory will be reused in several components.